src/test/ui/type-alias-impl-trait/collect_hidden_types.rs

```diff
@@ -0,0 +1,22 @@
+// aux-build:collect_hidden_types.rs
+use collect_hidden_types::Service;
+use std::future::Future;
+use std::pin::Pin;
+use std::task::Context;
+
+// build-pass
+
+// edition:2018
```

Comment:
I'm not sure we parse these (at least yet -- unless we already merged the compiletest PR Pietro was working on here)?

I'd rather they be at the top of the file regardless.

Reply:
We have many other such cases in our test suite. The comments only get ignored after the first item, possibly even allowed after use statements.